src/constant/links.tsx

```diff
@@ -117,3 +117,30 @@ export const footerLinks = {
     },
   ],
 };
+
+export const footerLinksOval = {
+  internal: [
+    {
+      label: "For Searchers",
+      href: "https://docs.oval.xyz/for-searchers",
+    },
+    {
+      label: "For Integrators",
+      href: "https://docs.oval.xyz/integration",
+    },
+    {
+      label: "Stats",
+      href: "https://stats.uma.xyz/",
```

Comment:
im wondering if this is hte right stats page to show, its based on mostly osnap and has no oval stats. perhaps one of these are better?
https://dune.com/risk_labs/aave-oev-via-liquidation-bonuses
or
https://dune.com/risk_labs/compound-v2-and-v3-liquidation-incentives

Reply:
actually might be better to just link to the docs: https://docs.oval.xyz/oev-data

Reply:
oh this is a mistake. this is from the uma homepage's footer 😬

Reply:
removing it entirely